docs/tutorial-2/multi-node.rst

```diff
@@ -0,0 +1,189 @@
+==========
+Multi-node
+==========
+
+This tutorial will show how to install MicroCeph on three machines, thereby
+creating a multi-node cluster.
+
+Ensure storage requirements
+---------------------------
+
+Three OSDs will be required to form a minimal Ceph cluster. This means that, on
+each of the three machines, one entire disk must be allocated for storage.
+
+The disk subsystem can be inspected with the :command:`lsblk` command. In this
+tutorial, the command's output on each machine looks very similar to what's
+shown below. Any output related to possible loopback devices has been
+suppressed for the purpose of clarity:
+
+.. code-block:: none
+
+   lsblk | grep -v loop
+
+   NAME   MAJ:MIN RM   SIZE RO TYPE MOUNTPOINTS
+   vda    252:0    0    40G  0 disk
+   ├─vda1 252:1    0     1M  0 part
+   └─vda2 252:2    0    40G  0 part /
+   vdb    252:16   0    20G  0 disk
+
+For the example cluster, each machine will use ``/dev/vdb`` for storage.
+
+Prepare the three machines
+--------------------------
+
+On each of the three machines we will need to:
+
+* install the software
+* disable auto-updates of the software
+
+Below we'll show these steps explicitly on **node-1**, which we'll call the
+primary node.
+
+Install the most recent stable release of MicroCeph:
+
+.. code-block:: none
+
+   sudo snap install microceph
+
+Prevent the software from being auto-updated:
+
+.. code-block:: none
+
+   sudo snap refresh --hold microceph
+
+Repeat the above three steps for node-2 and node-3.
+
+Prepare the cluster
+-------------------
+
+On **node-1** we will now:
+
+* initialis the cluster
+* create registration tokens
+
+Initialise the cluster with the :command:`cluster bootstrap` command:
+
+.. code-block:: none
+
+   sudo microceph cluster bootstrap
+
+Tokens are needed to join the other two nodes to the cluster. Generate these
+with the :command:`cluster add` command.
```

Comment:
Just for avoidance of doubt, could maybe add a note that the tokens will be different on the users' machine